libs/dal/src/repositories/job/job.entity.ts

```diff
@@ -21,7 +21,7 @@ export class JobEntity {
   _notificationId: string;
   subscriberId: string;
   _subscriberId: string;
-  _mergedDigestId?: string;
+  _mergedDigestId?: string | null;
```

Comment:
Now, we can remove assignments from _mergedDigestId. When a merged digest becomes a `main follower`, it's no longer merged, so it can be empty/null.